### ISSUE TITLE
In a Content Item's edit page, show widget views instead of edit cshtml

### ISSUE THREAD

**Issue by jackliDev** [OrchardCMS/OrchardCore#5930]
Let's say I have a Content Item containing other widgets.

When I go to Content -> Content Items -> Edit,
how can I show the widgets' view cshtml instead of edit cshtml?

I want to do this so that users can see what the end result will look like while constructing/modifying the content item.

**Comment by jtkech** [comment]
When editing a content item, try to click the `Preview` button to see in live what you are changing

**Comment by jackliDev** [comment]
Hi @jtkech we're actually looking for a programmatic change. 

We have a content item page called "Hub Page" that contains a widget called "Alert". If we go to Content Item -> Edit for Hub Page, we see the following: 
![image](https://user-images.githubusercontent.com/39414376/79148446-78e29080-7d93-11ea-9fbd-14daa2998c60.png)

If we "view" the page to see how it actually renders. It'll look like the following:
![image](https://user-images.githubusercontent.com/39414376/79148770-0e7e2000-7d94-11ea-9b8b-171997a276c6.png)

The circled section is our Alert widget.

Is there a way to show the final view result as above on the content item's Edit page so that the user can see what it looks like while modifying the content item?

**Comment by jtkech** [comment]
Did you try the **Preview** button at the bottom of the edit page?

You can then have 2 windows side by side, your edit view and on the right the preview in live.

**Comment by jackliDev** [comment]
I understand that Preview will allow the user to view the final result. However, for our UX, we specifically need to implement it so that widgets show their views on the content item edit page rather than being in edit mode.

**Comment by jtkech** [comment]
Ah okay, like a preview but not for the whole item, only for the contained widget you are editing

**Comment by jackliDev** [comment]
Yes, for content item edit page, we still want it to be an edit page for the content item overall, but for the widgets, we want those to render like preview

**Comment by ns8482e** [comment]
@jackliDev if you are using FlowPart - you can override FlowPart_Edit shane and set BuildEditor to false
https://github.com/OrchardCMS/OrchardCore/blob/669636359880be84f9321d8ca95abed2b91e7a25/src/OrchardCore.Modules/OrchardCore.Flows/Views/FlowPart.Edit.cshtml#L34

**Comment by sebastienros** [comment]
How would you edit the widgets if you don't render the editor but the end result?
